posthog/api/exports.py

```diff
@@ -258,7 +258,7 @@ def safely_get_queryset(self, queryset):
 
             # Add export format filter
             export_format_filter = self.request.query_params.get("export_format")
-            if export_format_filter and export_format_filter in ExportedAsset.SUPPORTED_FORMATS:
+            if export_format_filter and export_format_filter in ExportedAsset.get_supported_format_values():
```

Comment:
i'm interested why no failing tests anywhere for this
which would be nice to protect us in the future
but this solution is much nicer